tools/count_vale_errors.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
```

Comment:
you should add license on top of the file

Reply:
Done.